tools/taskcluster/upload_apk_nimbledroid.py

```diff
@@ -21,23 +21,15 @@ def uploadApk(apk,key):
 	response = requests.post(url, auth=(key, ''), headers=headers, files=apk, data=payload)
 
 	if response.status_code != 201:
-		print('Status:', response.status_code, 'Headers:', response.headers, 'Error Response:',response.json())
+		print('Error Response:',response.json())
 		exit()
```

Comment:
exit() is short for exit(0). You may want to use something like exit(1) here and then your shell script is not needed anymore.